core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -2217,7 +2254,9 @@
       </xs:enumeration>
       <xs:enumeration value="DEFAULT">
         <xs:annotation>
-          <xs:documentation>Use the default eviction listener thread policy (PIGGYBACK)</xs:documentation>
```

Comment:
This is all being removed in my PR as well. https://github.com/infinispan/infinispan/pull/7815/files#diff-0ada09d10de5ef5b0bdbe6db4bf95349L2211

Reply:
@wburns is this going to be removed in 10.1.x too?

Reply:
No

Reply:
same here @wburns keep in the doc PR for 10.1.x and remove in this doc PR?

Reply:
This configuration option is completely ignored for a long time, I don't even remember when.